src/code42cli/cmds/search/enums.py

```diff
@@ -1,21 +1,13 @@
+from code42cli.output_formats import OutputFormat
+
 IS_CHECKPOINT_KEY = "use_checkpoint"
 
 
-class OutputFormat:
+class SecurityDataOutputFormat(OutputFormat):
     CEF = "CEF"
-    JSON = "JSON"
-    RAW = "RAW-JSON"
-
-    def __iter__(self):
-        return iter([self.CEF, self.JSON, self.RAW])
-
-
-class AlertOutputFormat:
-    JSON = "JSON"
-    RAW = "RAW-JSON"
 
     def __iter__(self):
-        return iter([self.JSON, self.RAW])
+        return iter([self.TABLE, self.CSV, self.JSON, self.RAW, self.CEF])
```

Comment:
Output formats are not only restricted to `alerts/security-data` commands. They were also used elsewhere, so CEF would also be supported in those. I am ok with it, making sure everyone is on same page.

Naming it as `AlertOutputFormat` in that case might not suffice.

Reply:
CEF is a format that is only implemented for / only makes sense for security events -- it isn't a data format that can be applied to everything like JSON or CSV can, so we would want this to remain specific to security events.

Reply:
This is only used by File Events

Reply:
I can move it somewhere to be more obviois